cnf-certification-test/accesscontrol/suite.go

```diff
@@ -649,14 +694,17 @@ func testSysPtraceCapability(shareProcessPods []*provider.Pod) {
 		}
 		if !sysPtraceEnabled {
 			tnf.ClaimFilePrintf("Pod %s has process namespace sharing enabled but no container allowing the SYS_PTRACE capability.", put.String())
-			podsWithoutSysPtrace = append(podsWithoutSysPtrace, put.String())
+			nonCompliantObjects = append(nonCompliantObjects, testhelper.NewPodReportObject(put.Namespace, put.Name, "Non-compliant because it has process namespace sharing enabled but no container allowing the SYS_PTRACE capability", false))
```

Comment:
As we are appending objects to a non-compliant/compliant category adding "Non-compliant because..."/"Compliant because..." sounds a bit redundant to me.

Reply:
You are correct I shouldn't have to add that part of the string. I'll adjust where I use that.